sql/python/test_magic.py

```diff
@@ -40,7 +40,7 @@ def tearDown(self):
 
     def test_stander_sql(self):
         ret = ipython.run_cell_magic("sqlflow", "", self.select_statement)
-        ret_list = [r for r in ret.rows()]
+        ret_list = [r for r in ret.get(0).rows()]
```

Comment:
@typhoonzero I think we should change this behavior at `pysqlflow`. If `CompoundMessage` only contains one item with `TypeRows`, the display function should return the item itself, so that `Rows.__repr__` will display the data in table presentation in IPython Notebook.

Reply:
Hi @tonyyang-svail , I would like always using `list` as the returned value type of `display` function in pysqlflow, it looks like the behavior is more consistent. How do you think to merge this PR and talk about this in the future, I want to release the didi Docker image ASAP.

Reply:
@Yancey1989 @typhoonzero If the return type of `display` is a list, what would a user after typing the following in the Jupyter Notebook?
```
%%sqlflow
SELECT 1;
```

Reply:
The presentation looks quite different.

![Screen Shot 2019-09-27 at 3 50 39 PM](https://user-images.githubusercontent.com/29932814/65806497-a725b900-e13e-11e9-9ec9-4bde95a5b80d.png)